Updates PoW page with up-to-date info on block rewards, GPU mining

s-ben opened the pull request:
Removes out-of-date numbers on block reward. Removes advice to mine on video card, adding note that ASICs are out now, making GPU mining unlikely to be profitable. ( Closes #674 ).

jholdstock commented on docs/mining/proof-of-work.md:
```diff
@@ -7,21 +7,31 @@
-higher hashrates and therefore receive higher rewards.
+your computer to PoW mining. 
+
+PoW mined can be done using GPUs. However, ASICs that mine DCR are now available, 
```
```suggestion
PoW mining can be done using GPUs. However, ASICs that mine DCR are now available, 
```

jholdstock commented on docs/mining/proof-of-work.md:
```diff
@@ -7,21 +7,31 @@
+network. Each time a block is created (by a miner), which is then approved by 
+ticket holders, new coins are created. These new coins are called the block reward. 
+The block reward adjusts every 6,144 blocks (approximately 21.33 days) by reducing
+by a factor of 100/1011. 
```
```suggestion
by a factor of 100/101. 
```

jholdstock commented on docs/mining/proof-of-work.md:
```diff
@@ -7,21 +7,31 @@
 transactions and build the blocks that make up the blockchain in the Decred
-network. Each time a block is created (by a miner), about 30 new Decred coins
-are made. These coins are then split up as follows:
+network. Each time a block is created (by a miner), which is then approved by 
```
The 30% of the block reward which is allocated to the PoS voters is actually created regardless of whether they vote to approve or reject the block. If the 30% was not created, the voters would be incentivized to never reject a block.

The first answer under this video from @davecgh explains it well, it might actually be worth just copy and pasting part/all of this.
 
 https://www.reddit.com/r/decred/comments/66j4l4/decred_proof_of_stake_explained/

jholdstock replied:
Theres also a second reply from Dave under his first one with some more details.

s-ben replied:
I decided to not include further detail from @davecgh's post. I think that PoW miners may not want that level of detail on the PoW page - and it is included on other advanced pages. If you think that extra detail would be good to add here though, can work it in.

s-ben commented:
@jholdstock , thx for the review. Have addressed comments. Decided not to add more detail on what happens when blocks are rejected (see comment), but can add that in if you think the page will be better for it.
